CHANGES/1452.bugfix

```diff
@@ -0,0 +1,3 @@
+Populate a context for use in v2 serializers
+Add name to route to satisfy v2 ``get_latest_version`` href
+Copy logic from ``UnpaginatedCollectionVersionSerializer.get_download_url`` to build a valid ``download_url`` for the v2 API
```

Comment:
We write these changelogs in the past tense, also please add punctuation.
Given that this changelog is what we tell the user, can you write something like "Fixed some bugs in the galaxy v2 api, where ...", so users experiencing the issue can relate?